parallel-install/pkg/deployment/core.go

```diff
@@ -157,8 +165,8 @@ func removeFromComponentList(cl *components.ComponentList, componentNames []stri
 	}
 }
 
-func registerOverridesInterceptors(o *OverridesBuilder) {
+func registerOverridesInterceptors(kubeClient kubernetes.Interface, o *OverridesBuilder, retryOptions []retry.Option, log logger.Interface) {
 	//hide certificate data
-	o.AddInterceptor([]string{"global.domainName", "global.ingress.domainName"}, &DomainNameOverrideInterceptor{})
+	o.AddInterceptor([]string{"global.domainName", "global.ingress.domainName"}, NewDomainNameOverrideInterceptor(kubeClient, retryOptions, log))
```

Comment:
From what I can see in the code, the logic (including retries etc.) is performed twice, because we are providing two keys? It would be nice to perform the k8s related operations only once.

Reply:
idk it's worth investing in that, cause soon we will get rid of ingress.domainName